website/content/docs/configuration/listener/unix.mdx

```diff
@@ -0,0 +1,69 @@
+---
+layout: docs
+page_title: Unix - Listeners - Configuration
+description: |-
+  The Unix listener configures Vault to listen on the specified unix domain socket.
```

Comment:
```suggestion
  The Unix listener configures Vault to listen on the specified Unix domain socket.
```

Reply:
Unix is a brand name, like Nike. But not UNIX because we aren't using the trademark